blockchain/v2/reactor.go

```diff
@@ -382,7 +385,10 @@ func (r *BlockchainReactor) demux(events <-chan Event) {
 				r.processor.send(event)
 				r.scheduler.stop()
 			case scSchedulerFail:
-				r.logger.Error("Scheduler failure", "err", event.reason.Error())
+				if scSchedulerFailErr != event.reason {
+					r.logger.Error("Scheduler failure", "err", event.reason.Error())
+					scSchedulerFailErr = event.reason
+				}
```

Comment:
This is so we just don't keep logging the same thing right?

Reply:
yep